### DISCUSSION TITLE
Css scrollbar

### DISCUSSION THREAD

**Discussion by vaniprasad338** [xyflow/xyflow#4006] [Q&A]
Can we use the native css scrollbar in the react-flow library? I tried using it, but the edges are acting weirdly.

**Comment by bcakmakoglu** [comment]
It is kinda possible but might not result in the greatest experience.

[Here's an example.](https://codesandbox.io/p/sandbox/recursing-curie-wnt36v?file=%2Fsrc%2FApp.tsx%3A37%2C1)
